### DISCUSSION TITLE
KMS: Need Alternative

### DISCUSSION THREAD

**Discussion by Reflux0301** [HotCakeX/Harden-Windows-Security#370] [Q&A]
Hey there, I use a KMS server and activate Windows EUDs that way, I understand VBScript is deprecated and as such, `slmgr.vbs` and `ospp.vbs` will be unusable. I'd like to keep my KMS infra running but what's the best way to activate without using VBScript (if it's even possible).

Thanks.

**Comment by HotCakeX** [comment]
Hi,
I'm planning on adding a section to the Harden Windows Security user interface where you can choose individual optional features, capabilities, apps etc. to be removed or added, instead of adding/removing them all at the same time using the [Optional Windows Features category](https://github.com/HotCakeX/Harden-Windows-Security?tab=readme-ov-file#optional-windows-features).

Until then you can simply use the Windows settings to add the VBS feature back if you removed it.

<br>

<img width="1081" alt="image" src="https://github.com/user-attachments/assets/c614ba67-b55b-4870-8411-7047fbc7152e">
